cowprotocol/cowswap
# feat(twap): scheduled order warnings

# Summary

Part of #2880 
Depends on https://github.com/cowprotocol/cowswap/pull/2876

![image](https://github.com/cowprotocol/cowswap/assets/43217/7409d7fa-8973-43d1-b729-f0387902c1a0)

- Updates Low balance warning
- Updates Low allowance warning

# To Test

1. Place twap order
2. Remove balance
* Should show low balance warning
* Warning should be specific to the scheduled order
3. Add balance, remove approval
* Should show low allowance warning
* Warning should be specific to the scheduled order
4. Repeat with LIMIT order
* Warnings should be same as existing warning
